variants/pundit/template.rb

```diff
@@ -29,8 +29,35 @@
   private
 
   def user_not_authorized
-    flash[:alert] = I18n.t("authorization.not_authorized")
-    redirect_to(request.referer || root_path)
+    # This application uses the Rails default of increasing numeric IDs for
+    # records in the database.
+    #
+    # This means there are security implications to how you inform a user that
+    # they are not authorized.  If this app returns a different error message
```

Comment:
```suggestion
    # they are not authorized. If this app returns a different error message
```

🐘 got a double space here